app/presenters/vendor_api/single_application_presenter.rb

```diff
@@ -10,6 +10,7 @@ def as_json
         id: application_choice.id.to_s,
         type: 'application',
         attributes: {
+          application_reference: application_form.support_reference,
```

Comment:
Sorry to be late to this party, but can we rename this to "support reference"? It would be consistent with the field name, and avoid confusion with the `ApplicationReference` model.

Reply:
sure, will do!